src/ship/passiveSensors.js

```diff
@@ -19,6 +20,6 @@ export default class PassiveSensors{
             readings.push(newReading);
         }
 
-        return readings;
+        return new response(response=readings, success=true);
```

Comment:
Same consideration as activeSensors. Should ensure that documentation reflects correct `response` payload type (either current array value, or change to object)